### PR TITLE
Added absolute result to result column

### PR DESCRIPTION
**Pull request by eatsleepcoderepeat-gl**
Added absolute result to result column for the positions table on the dashboard

### CODE REVIEW

File: cryptohopper-dashboard-watchlist.user.js [hunk 1]
```diff
@@ -35,6 +35,9 @@ var REMOVE_HOPPIE = true;
 // and a "buy" indicator where your last purchase was with the average rate.
 var SHOW_BUY_RATE_IN_TRADING_VIEW = true;
 
+// Adds an absolute value to the Result column on the dashboard
+var ADD_ABSOLUTE_RESULT = true;
```
Comment (markrickert) [inline cryptohopper-dashboard-watchlist.user.js:39]: Thanks for feature flagging this! Eventually i'd like these values to be configurable for an average user with a div element on screen.

Comment (eatsleepcoderepeat-gl) [reply cryptohopper-dashboard-watchlist.user.js:39]: This was exactly my thinking as well. An unobtrusive pop-out menu from the side of the page is what I'm currently thinking, a little like the chat available on the site.

**Comment by markrickert** [comment]
OK, so this is awesome. It was a starting point for me to figure out how to hook into the websockets data and parse that. using this method by just passively listening for data changes instead of a bunch of `setTimeout`s that make the screen flash and take up cpu cycles. I'm really stoked to be able to use the websockets here! Check out this branch and let me know what you think! https://github.com/markrickert/cryptohopper-dashboard-watchlist/tree/add-absolute-numbers

**Comment by markrickert** [comment]
I've split this project into multiple userscript files now so each feature is sandboxed and added this code as a new userscript with your authorship attributed 👍  Thanks! Closing this PR.

**Pull request closed without merging**